Automattic/pocket-casts-ios
# Playback 2023: animates Year over Year

| 📘 Part of: #1142 |
|:---:|

Adds an animation to the Year over Year story:


https://github.com/Automattic/pocket-casts-ios/assets/7040243/97297e9e-7d02-406c-800f-e230b6570f97



## To test

1. Log in to an account that has a few episodes listened this year and last year
3. Go to Profile and tap the End of Year card
4. Navigate to the Year over Year story
5. ✅ The bars should animate and the text should fade-in
6. Tap "Share this Story" > Save Image
7. ✅ Go to Photos and the story should be there correctly

## Checklist
